reciprocalspaceship/__init__.py

```diff
@@ -18,21 +21,21 @@ def getVersionNumber():
 # Add support for MTZ data types:
 # see http://www.ccp4.ac.uk/html/f2mtz.html
 from .dtypes import (
```

Comment:
Do you mind reverting this file back to original and only adding `read_crystfel` to the top-level namespace? I know the file isn't quite "conventionally" formatted, but I find the aligned comments most clear

Reply:
Sure, do it. I just have `yapf` on by default, which messed already existing files.